### PR TITLE
Add Cadent Aperture Mx Alias

### PR DESCRIPTION
**Pull request by AntoxaAntoxic**

### CODE REVIEW

File: src/main/resources/bidder-config/emxdigital.yaml [hunk 1]
```diff
@@ -12,7 +15,7 @@ adapters:
       supported-vendors:
       vendor-id: 183
     usersync:
-      cookie-family-name: emx_digital
+      cookie-family-name: emxdigital
```
Comment (SerhiiNahornyi) [inline src/main/resources/bidder-config/emxdigital.yaml:18]: why cookie-family-name was changed?

File: src/main/resources/bidder-config/emxdigital.yaml [hunk 2]
```diff
@@ -1,8 +1,11 @@
 adapters:
   emxdigital:
     endpoint: https://hb.emxdgt.com
+    aliases:
+      cadentaperturemx:
```
Comment (SerhiiNahornyi) [inline src/main/resources/bidder-config/emxdigital.yaml:5]: Also from go code, looks like thay expect 'cadent_aperture_mx'

**Review by SerhiiNahornyi — CHANGES_REQUESTED** [review]

File: src/main/java/org/prebid/server/spring/config/bidder/EmxDigitalConfiguration.java [hunk 3]
```diff
@@ -19,7 +19,7 @@
 @PropertySource(value = "classpath:/bidder-config/emxdigital.yaml", factory = YamlPropertySourceFactory.class)
 public class EmxDigitalConfiguration {
 
-    private static final String BIDDER_NAME = "emx_digital";
+    private static final String BIDDER_NAME = "emxdigital";
```
Comment (SerhiiNahornyi) [inline src/main/java/org/prebid/server/spring/config/bidder/EmxDigitalConfiguration.java:22]: Why this was changed?

File: src/main/resources/bidder-config/emxdigital.yaml (hunk 1 above)
Comment (AntoxaAntoxic) [reply src/main/resources/bidder-config/emxdigital.yaml:18]: yeah, probably shouldn't

File: src/main/resources/bidder-config/emxdigital.yaml (hunk 2 above)
Comment (AntoxaAntoxic) [reply src/main/resources/bidder-config/emxdigital.yaml:5]: I thought we could go with our own naming, couldn't we? We still have a helper endpoint to find all the bidders and aliases

File: src/main/java/org/prebid/server/spring/config/bidder/EmxDigitalConfiguration.java (hunk 3 above)
Comment (AntoxaAntoxic) [reply src/main/java/org/prebid/server/spring/config/bidder/EmxDigitalConfiguration.java:22]: I spot some inconsistencies and failures related to the bidder name, yaml and json files names, so I decided to rename the bidder for good

File: src/test/resources/org/prebid/server/it/test-application.properties [hunk 4]
```diff
@@ -150,8 +150,9 @@ adapters.edge226.enabled=true
 adapters.edge226.endpoint=http://localhost:8090/edge226-exchange
 adapters.emtv.enabled=true
 adapters.emtv.endpoint=http://localhost:8090/emtv-exchange
-adapters.emxdigital.enabled=true
-adapters.emxdigital.endpoint=http://localhost:8090/emx_digital-exchange
+adapters.emx_digital.enabled=true
+adapters.emx_digital.endpoint=http://localhost:8090/emx_digital-exchange
+adapters.emx_digital.aliases.cadent_aperture_mx.enabled=true
```
Comment (CTMBNara) [inline src/test/resources/org/prebid/server/it/test-application.properties:155]: Please, use original path that is present in .yaml configuration file.
`adapters.emxdigita. ...`

**Review by CTMBNara — CHANGES_REQUESTED** [review]

**Review by SerhiiNahornyi — APPROVED** [review]

**Pull request merged**